content/companies/02-gatsby.md

```diff
@@ -18,6 +18,7 @@ excerpt: Know more about Gatsby's interview process ➡️
 |      **Scripted Questions?**       |  No   |
 | **Typical DS and Algo Questions?** |  No   |
 |          **Live Coding?**          |  Yes  |
+|       **Asked Current CTC?**       |  No   |
```

Comment:
Once you make that template change suggested above you can remove this and I’ll create a new PR to match all the blogs with the template

Reply:
Yes, I  misunderstood things last time. 
I have pushed updated changes, created a heavily commented markdown template at `src/template.md` and updated README as well.

I think now you can drop appendix section in other posts, they seems like guiding how you should write one, that part is moved to template comments.